modules/checking-images.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * cli/checking-policy-compliance.adoc
+:_content-type: PROCEDURE
+[id="checking-images_{context}"]
+= Checking images
+
+.Procedure
+
+* Run the following command to check the build-time violations of your security policies in images:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl "$ROX_CENTRAL_ADDRESS" image check --image=_<image_name>_
```

Comment:
The correct variable is `ROX_ENDPOINT` and we do not need to set this explicitly here.

Since we initially documented it together with the `ROX_API_TOKEN` we can drop this here.

If we want to keep it, let's use it like:
```bash
export ROX_ENDPOINT=<host:port>
roxctl image check --image=<image name>
```